ReferenceLibrary/src/main/java/referencelibrary/util/FileUtil.java

```diff
@@ -37,20 +37,24 @@ public static String Read(String fileName) {
         try (BufferedReader br = new BufferedReader(new FileReader(fileName))) {
             StringBuilder sb = new StringBuilder();
             String line = br.readLine();
```

Comment:
Olisi mukavaa jos tämäkin rivi olisi uuden metodin sisällä. Tällöin olisi myös metodissa tarve yhtä vähemmälle parametrille ja se tekisi kokonaan lukemisen readerista builderiin.